oak-core/src/test/java/org/apache/jackrabbit/oak/query/AbstractQueryTest.java

```diff
@@ -108,17 +108,21 @@ protected void createTestIndexNode() throws Exception {
         root.commit();
     }
 
-    protected static Tree createTestIndexNode(Tree index, String type)
-            throws Exception {
+    protected  static Tree createTestIndexNode(String indexName, Tree index, String type) {
         Tree indexDef = index.addChild(INDEX_DEFINITIONS_NAME).addChild(
-                TEST_INDEX_NAME);
+                indexName);
         indexDef.setProperty(JcrConstants.JCR_PRIMARYTYPE,
                 INDEX_DEFINITIONS_NODE_TYPE, Type.NAME);
         indexDef.setProperty(TYPE_PROPERTY_NAME, type);
         indexDef.setProperty(REINDEX_PROPERTY_NAME, true);
         return indexDef;
     }
 
+    protected static Tree createTestIndexNode(Tree index, String type)
+            throws Exception {
```

Comment:
I guess this can be removed

```suggestion
    protected static Tree createTestIndexNode(Tree index, String type) {
```